spec/models/book_spec.rb

```diff
@@ -22,4 +22,14 @@
       expect(books.map(&:title)).to eq ["The it"]
     end
   end
+
+  describe "#find" do
+    it "should find a book by slug title" do
+      book = create(:book, title: 'The book')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.